cmd/deploy/deploy.go

```diff
@@ -336,6 +357,50 @@ func setManifestEnvVars(service, reference string) error {
 	return nil
 }
 
+func deployDependency(ctx context.Context, name string, dependency *model.Dependency) error {
```

Comment:
Can we use `pipeline.ExecuteDeployPipeline(ctx, remoteOpts)` so we don't need to create new code?

Reply:
The only thing that might not be the same is the logs such as `Dependency '%s' scheduled for deployment` but maybe we can create a new option to have those logs